livre/eq/settings/VolumeSettings.h

```diff
@@ -35,33 +36,37 @@ class VolumeSettings : public co::Serializable
 {
 public:
 
-    /**
-     * @brief VolumeSettings constructor.
-     */
-    VolumeSettings( );
+    /** VolumeSettings constructor. */
+    VolumeSettings();
 
-    /**
-     * @brief reset Resets the information.
-     */
-    void reset( );
+    /** reset Resets the information. */
+    void reset();
 
     /**
-     * @brief setModelFileName Sets the models source uri.
+     * Sets the models source uri.
      * @param uri The uri of the source data.
      */
     void setURI( const std::string& uri );
 
+    /** @return Returns the uri of the source data. */
+    const std::string& getURI( ) const;
+
     /**
-     * @return Returns the uri of the source data.
+     * Sets the accumulated data range for the volume.
+     * @param range of the data source.
      */
-    const std::string& getURI( ) const;
+    void setDataSourceRange( const Vector2f& range );
+
+    /** @return Returns the accumulated data range for the volume. */
+    const Vector2f& getDataSourceRange() const;
```

Comment:
I feel this should be called differently, like getHistogramRange or getGlobalDataRange, in any case remove Source, I don't think it's needed.